src/core/scrobbler/listenbrainz/listenbrainz-scrobbler.ts

```diff
@@ -346,7 +335,9 @@ export default class ListenBrainzScrobbler extends BaseScrobbler<'ListenBrainz'>
 		// NOTE: Use 'same-origin' credentials to fix login on Firefox ESR 60.
 		const promise = fetch(url, {
 			method: 'GET',
+			// #v-ifndef VITE_FIREFOX
```

Comment:
ifndef is for every browser except for firefox, i think you mightve meant ifdef?

Reply:
Good catch :see_no_evil: